### ISSUE TITLE
Application error

### ISSUE THREAD

**Issue by mds-vsu** [GACWR/OpenUBA#43]
Hi, OpenUBA development team!

I'm trying to run openuba locally on Ubuntu 20.04.
Could you please advise how to fix: i have an issue running the application, looks like some problems related to connecting to dev server
I had provided server and application logs and the screenshot from firefox 
Will appreciate your help, thank you

![image](https://user-images.githubusercontent.com/71710818/97017700-d47ea980-1580-11eb-8502-6328d215f405.png)

[logs.zip](https://github.com/GACWR/OpenUBA/files/5430074/logs.zip)

**Comment by Jovonni** [comment]
Hey @mds-vsu , these doesn't have anything to do with connecting to the dev server. This warning is purely related to the DOM hierarchy within react.

**see:**
https://stackoverflow.com/questions/55625431/warning-validatedomnesting-a-cannot-appear-as-a-descendant-of-a

This has been on our radar, and is a simple fix, just haven't gotten to fixing it yet. It doesn't hurt development though, as it is only within react.

Will label it, and keep open until we address it. I am glad to see you can successfully build the frontend. Please either let us know how we can make the documentation better, or if you want to create a pull request and help out. 

If you want to look more into this, look at the SystemLog context consumer (this is using the react context api, and you will see the provider in the same file as well):
https://github.com/GACWR/OpenUBA/blob/ae56fe29d2348845f93979de2999f73c7c121f11/interface/src/Components/Common/SystemLog/SystemLog.js#L13-L28

This context implementation is setup to feed frontend related display information via context. Thanks for posting the issue, and feel free to join our discord (if you need some quicker help). We are still actively building on our first release.

:)

**Comment by mds-vsu** [comment]
@Jovonni thank you for rapid reply.
I'm working on master thesis with UEBA topic and want to try your software. Will be glad to contribute with something.
I still have the feeling that frontend displays just placeholders.
I'm attached some screenshots of other application sections

I got this when trying to switch to "Models"
![image](https://user-images.githubusercontent.com/71710818/97077903-17409000-161a-11eb-8ae4-14abfed1e268.png)

Settings, Cases and Anomalies just displays a Header, like "Settings page", "Anomalies page", etc..
![image](https://user-images.githubusercontent.com/71710818/97077931-68508400-161a-11eb-9bf3-fa979aa023a1.png)

**Comment by Jovonni** [comment]
Yes, all configuration in this version is done via the actual source code. We haven't wired up all of the pages yet in the code you have. Our current release is further on that.

Don't rely on the UI to change anything right now. The only page that is started to be connected to backend is the models page, as that is receiving data via the context API.

Still working on that. 

Regarding that renderer error, can you describe the behavior leading up to that?

We can check that API call it is referencing.

The model page is pulling in the two mode groups in the configuration file.

For testing, we have been customizing everything via the source code, not UI. More on the UI connecting to the backend soon, but it isn't needed to proceed through the workflow.

**Comment by mds-vsu** [comment]
@Jovonni Got it, thank you for reply!
